mtc.go

```diff
@@ -2039,3 +2044,68 @@ func (tai *TrustAnchorIdentifier) unmarshal(s *cryptobyte.String) error {
 	tai.Issuer = oid
 	return nil
 }
+
+// VerifyOpts includes parameters for verifying a BikeshedCertificate.
+type VerifyOptions struct {
+	// SignedValidityWindow is a signed validity window that covers the
+	// certificate. It is the caller's responsibility to verify the
+	// validity window's signature using the CA parameters.
+	SignedValidityWindow *SignedValidityWindow
+
+	// CA includes the parameters of the CA that issued the batch
+	// containing the certificate.
+	CA *CAParams
+
+	// CurrentTime is used to to check if the certificate has expired.
+	CurrentTime time.Time
+}
+
+// Verify is used to verify that a BikeshedCertificate is covered by a signed
+// validity window. It is the caller's responsibility to verify the validity
+// window's signature. An error indicates that the certificate does not belong
+// to a batch in the validity window or that the certificate is otherwise
+// invalid.
+func Verify(cert *BikeshedCertificate, opts VerifyOptions) error {
+	p, ok := cert.Proof.(*MerkleTreeProof)
+	if !ok {
+		return fmt.Errorf("Expected MerkleTreeProof; got %t", cert.Proof)
+	}
+
+	// Check if the certificate has expired.
+	if !opts.CurrentTime.Before(cert.Proof.NotAfter()) {
+		return fmt.Errorf("Certificate has expired (%v)", p.NotAfter().UTC())
+	}
```

Comment:
There's no concept of "not before" correct?

Reply:
Correct